test/integration/lib/foxboxHelper.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var helper = (function() {
+
+    function removeDB() {
+      var fs = require('fs');
+      var filePath = process.env.HOME + 
+      '/.local/share/foxbox/users_db.sqlite' ; 
+      fs.exists(filePath,function(exists){
+        if(exists) {
+          fs.unlinkSync(filePath);
```

Comment:
Using a sync method is usually a bad practice in Node, as it block the main (and only) thread. I think we can return a promise to make `removeDb` async.